apps/portal/src/app/views/notifications/json-dialog/json-dialog.component.html

```diff
@@ -0,0 +1,4 @@
+<p-dialog header="JSON Data" [(visible)]="visible" [draggable]="false">
```

Comment:
Make it such that the width of the dialog is fixed instead of varying with content. See if it looks good if not then keep a appropriate max width instead.

As  a improvement see if there is any component for copy to clipboard, but dont spend too much time on it.

Reply:
Fixed width for this block in this commit : https://github.com/OsmosysSoftware/osmo-notify/pull/81/commits/3080483eccb3b22f7922d901461414c45cd6dfb8

Tried looking for alternative componenet, not able to find any.

Reply:
@PrinceYadav2 
Can you add updated Screenshots.
Also keep the title at the center for now as its at the extreme left right now and not looking good.

Reply:
Updated the screenshots as per latest view.

Reply:
@PrinceYadav2  I see the main title(Osmo Notify Portal) is still to the left

Reply:
I had to add a bit of styling to maintain the view, Noticed bootstrap was cached in previous version.

I have updated the latest screenshots.

Changes are done in this commit: https://github.com/OsmosysSoftware/osmo-notify/pull/81/commits/0637a8c2bc051ec8225707c1ea1f9d00d20c207a